Info-Orbs/src/widgets/clockWidget.cpp

```diff
@@ -17,7 +17,8 @@ void ClockWidget::setup() {
 }
 
 void ClockWidget::draw(bool force) {
-    bool showFirstDigit = FORMAT_24_HOUR || (!FORMAT_24_HOUR && m_hourSingle >= 10);
+    GlobalTime* time = GlobalTime::getInstance();
+    bool showFirstDigit = FORMAT_24_HOUR || (!FORMAT_24_HOUR && time->isPM());
```

Comment:
this is wrong. I think the first version is correct

Reply:
Yup, you're right. Got my changes mixed up.